src/datatable/DataTable.tsx

```diff
@@ -183,6 +183,8 @@ export default function DataTable(props: IProps) {
 			}
 			setColumnTypes(newColumnTypes);
 			setParsedRows(props.rows);
+		} else {
+			setRows(props.rows);
```

Comment:
Should this be calling `setParsedRows(...)` also?

Reply:
yup